test/test-najax.js

```diff
@@ -1,9 +1,15 @@
-/* globals describe beforeEach it */
+/* globals describe beforeEach afterEach it */
 var najax = require('../lib/najax.js')
 var expect = require('chai').expect
 var nock = require('nock')
 var zlib = require('zlib')
 
+// prevent cross-test or cross-describe contamination by
+// globally clearing nock after each test
+afterEach(function () {
+  nock.cleanAll()
+})
+
```

Comment:
I elected to do this globally since we were dealing with contamination between describe blocks.